designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -522,11 +525,27 @@ internal fun ContentDrawScope.render(
 
     fun getPaths(
         path: List<com.android.designcompose.serdegen.Path>,
-        stroke: List<com.android.designcompose.serdegen.Path>
+        stroke: List<com.android.designcompose.serdegen.Path>,
+        vectorSize: Optional<List<Float>>,
```

Comment:
Nit: Why not use a nullable?

Reply:
Just because in Rust, this was an Option<(f32, f32)> and serde generates this into Optional<List<Float>>